superset-frontend/src/components/Menu/SubMenu.tsx

```diff
@@ -195,69 +192,57 @@ const SubMenuComponent: React.FunctionComponent<SubMenuProps> = props => {
     return () => window.removeEventListener('resize', resize);
   }, [props.buttons]);
 
-  const offset = props.name ? headerSize : 0;
-
   return (
     <StyledHeader>
       <Row className="menu" role="navigation">
-        {props.name && (
-          <Col md={offset} xs={24}>
-            <div className="header">{props.name}</div>
-          </Col>
-        )}
-        <Col md={16 - offset} sm={24} xs={24}>
-          <Menu mode={showMenu} style={{ backgroundColor: 'transparent' }}>
-            {props.tabs &&
-              props.tabs.map(tab => {
-                if ((props.usesRouter || hasHistory) && !!tab.usesRouter) {
-                  return (
-                    <Menu.Item key={tab.label}>
-                      <li
-                        role="tab"
-                        data-test={tab['data-test']}
-                        className={
-                          tab.name === props.activeChild ? 'active' : ''
-                        }
-                      >
-                        <div>
-                          <Link to={tab.url || ''}>{tab.label}</Link>
-                        </div>
-                      </li>
-                    </Menu.Item>
-                  );
-                }
-
+        {props.name && <div className="header">{props.name}</div>}
+        <Menu mode={showMenu} style={{ backgroundColor: 'transparent' }}>
+          {props.tabs &&
+            props.tabs.map(tab => {
+              if ((props.usesRouter || hasHistory) && !!tab.usesRouter) {
                 return (
                   <Menu.Item key={tab.label}>
                     <li
-                      className={cx('no-router', {
-                        active: tab.name === props.activeChild,
-                      })}
                       role="tab"
+                      data-test={tab['data-test']}
+                      className={tab.name === props.activeChild ? 'active' : ''}
                     >
-                      <a href={tab.url} onClick={tab.onClick}>
-                        {tab.label}
-                      </a>
+                      <div>
```

Comment:
Is this `div` required?

Reply:
So the diff looks weird this is original business logic. I did not add anything.